servlet-galleon-pack/src/main/resources/feature_groups/domain-profile.xml

```diff
@@ -7,6 +7,8 @@
                 <param name="handlers" value="[&quot;FILE&quot;]"/>
             </include>
             <exclude spec="subsystem.logging.console-handler"/>
+            <exclude spec="subsystem.discovery"/>
+            <exclude spec="subsystem.jmx.remoting-connector.jmx"/>
```

Comment:
Is there a reason we're excluding these now?

Reply:
Thre reasons were described before for the legacy variant. We should not include `<remoting-connector/>` declaration in the domain.xml file for jmx subsystem.